Raise on load for the sequence data

ttung opened the pull request:
### Description
We want to raise an error if we ever erroneously load the sequence data because it is expensive to load erroneously.

This is a new feature of sqlalchemy 1.4, which we are bringing in with #189 

### Test plan
Add a test to verify that the `raiseload` does the expected thing.

phoenixAja reviewed (approved):
this is very nifty! 👍

Commit:
Raise on load for the sequence data

### Description
We want to raise an error if we ever erroneously load the sequence data because it is expensive to load erroneously.

This is a new feature of sqlalchemy 1.4.

### Test plan
Add a test to verify that the `raiseload` does the expected thing.
